Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    decanter (3.6.0)
+    decanter (3.7.0)
```

Comment:
Decanter is at `3.5.1` right now, so you can keep this version at `3.6.0` (you updated it on this PR in April)

Reply:
OPE (* midwestern stephen voice *). fixed.